shared/chat/conversation/normal/index.desktop.tsx

```diff
@@ -69,6 +69,7 @@ class Conversation extends React.PureComponent<Props> {
             )}
             {this.props.showLoader && <Kb.LoadingLine />}
           </Kb.Box2>
+          {/* TODO: put the block buttons in here */}
```

Comment:
do we have a ticket for that? or is it to be done in this PR?

Reply:
in this PR. I'm doing it today, I wanted to put this up in case other stuff depended on it.